skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerLogging.scala

```diff
@@ -38,6 +38,30 @@ object ConsumerLogging {
         } yield a
       }
 
+      // TODO RebalanceListener1 implement logging - https://github.com/evolution-gaming/skafka/issues/127
```

Comment:
please at least add same logging as in other `subscribe` methods

Reply:
good idea, thanks, fixed in c7d2f03c